src/reducers/catReducer.js

```diff
@@ -0,0 +1,22 @@
+import initalState from './initialState';
+import { GET_GIF_SUCCESS, GET_GIF_FAIL, GET_GIF_LOADING } from '../actions/actionTypes';
+
+export default function catReducer(state = initalState.Cats, { type, payload }) {
+    let newState = { ...state };
+    switch (type) {
+        case GET_GIF_SUCCESS:
+            console.log(payload);
+
+            newState["secondCat"] = payload.response.data.data.images.fixed_height;
```

Comment:
you're only ever updating and fetching from `secondCat` - just rename this variable to be "catGif" everywhere